doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-rule-based-proc.adoc

```diff
@@ -0,0 +1,25 @@
+[id='test-scenarios-coverage-report-rule-based-proc']
+= Generating coverage reports for rule-based test scenarios
+
+In rule-based test scenarios, the *Coverage Report* tab contains the detailed information about the following:
+
+* Number of available rules
+* Number of fired rules
+* Percentage of fired rules
+* Percentage of executed rules represented as a pie chart
+* Number of times each rule has executed
+* The rules that are executed for each defined test scenario
+
+Follow the procedure to generate a coverage report for rule-based test scenarios:
+
+.Prerequisites
+* The rule-based test scenario template are created for the selected test scenario. For more information about creating rule-based test scenarios, see xref:test-designer-create-test-template-rule-based-proc[].
+* The individual test scenarios are defined. For more information about defining a test scenario, see xref:test-designer-test-scenario-definition-proc[].
++
+NOTE: To generate the coverage report for rule-based test scenario, you must create at least one rule.
+
+.Procedure
+. Open the rule-based test scenarios in the test scenario designer.
+. Run the defined test scenarios.
+. Click *Coverage Report* on the right of the test scenario designer to display the test coverage statistics.
+. Optional: To download the test scenario coverage report, Click *Download report*.
```

Comment:
I have a suggestion to keep the optional the same as at 3.4.
Like "Optionally, to download the test scenario coverage report, Click *Download report*.

Reply:
The correct format to use in the document according to IBM style guide is **Optional:**instead of **Optionally**. I will change it in the **section 3.4** or wherever it is used. Thank you for this.